pkgs/by-name/gi/git-blame-someone-else/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "git-blame-someone-else";
+  version = "0";
+
+  src = fetchurl {
+    url = "https://github.com/jayphelps/git-blame-someone-else/archive/refs/heads/master.zip";
+    hash = "sha256-jZkUKv2SV28wsM18tCqNxoCZmLxdYH2Idh9RLibH2yA=";
+  };
```

```suggestion
  fetchFromGitHub,
}:

stdenv.mkDerivation (finalAttrs: {
  pname = "git-blame-someone-else";
  version = "0";

  src = fetchFromGitHub {
    owner = "jayphelps";
    repo="git-blame-someone-else";
    rev="master";
    hash = "sha256-jZkUKv2SV28wsM18tCqNxoCZmLxdYH2Idh9RLibH2yA=";
  };
```
using the appropriate git fetcher is always preferable.

---

are you kidding me git? i can only commit one?